src/simulation/monsters/low/Minotaur.ts

```diff
@@ -0,0 +1,40 @@
+import LootTable from '../../../structures/LootTable';
+import SimpleMonster from '../../../structures/SimpleMonster';
+import { GemTable } from '../../subtables/RareDropTable';
+
+export const MinotaurTable = new LootTable({ limit: 101 })
+	.every('Bones')
+
+	.tertiary(50, 'Ensouled minotaur head')
+	.tertiary(60, 'Clue scroll (beginner)')
+	.tertiary(101, 'Clue scroll (easy)')
+
+	/* Weapons and armour */
+	.add('Iron arrow', [5, 14], 10)
+	.add('Bronze spear', 1, 11)
+	.add('Bronze full helm', 1, 11)
+	.add('Bronze arrow', [3, 6], 6)
+	.add('Bronze dagger', 1, 7)
+
+	/* Runes */
+	.add('Mind rune', 1, 6)
+
+	/* Other */
+	.add('Coins', [2, 84], 16)
+	.add('Tin ore', 1, 13)
+	.add('Rune essence', 5, 6)
```

Comment:
```suggestion
```

Reply:
i still think this should be removed because no other monster that i know of drops both types of ess